build-image/builder.sh

```diff
@@ -43,6 +46,22 @@ if [ "$(cat /tmp/go-build/build_master)" == "${short_commit}" ]; then
   exit 130
 fi
 
+log "Verifying tag signatures..."
+for tag in ${tags}; do
+  if ( LANG=C git tag --verify ${tag} 2>&1 | grep "Good signature" ); then
+    LANG=C git tag --verify ${tag} 2>&1 | grep "gpg:" > /tmp/go-build/.signature_${tag}
```

Comment:
I was wondering how you were loading your web of trust, but it looks like [Git doesn't support that yet][1].  In any case, it should really be the downloader's web of trust, so passing the signature back to them for them to verify it locally would let them ensure that gobuilder wasn't being fooled by an impersonator's signature (although they'd still have to trust gobuilder to be dealing honestly).

[1]: http://mikegerwitz.com/papers/git-horror-story#automate